quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -842,7 +844,7 @@ class QuotationSpec extends Spec {
       }
       "abritrary" in {
         val q = quote(lift(String.valueOf(1)))
-        q.liftings.`java.this.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
+        //q.liftings.`java.lang.String.valueOf(1)`.value mustEqual String.valueOf(1)
```

Comment:
`this` is required by 2.11 and fail without, opposite to 2.12 which fails with. 
How should we handle that? Keep scala-2.1* different test directories or something like that. Any ideas?